google/cloud/dataproc_spark_connect/session.py

```diff
@@ -385,9 +385,9 @@ def _get_dataproc_config(self):
                     )
                 }
             if "COLAB_NOTEBOOK_ID" in os.environ:
-                dataproc_config.labels["colab-notebook-id"] = os.environ[
-                    "COLAB_NOTEBOOK_ID"
-                ]
+                dataproc_config.runtime_config.properties[
+                    "colab_notebook_id"
```

Comment:
Ideally we will add a new field for this, but for now maybe something like this will do:
```
dataproc.owner.type=COLAB_NOTEBOOK
dataproc.owner.name=os.environ["COLAB_NOTEBOOK_ID"]
```